vendor/vendor.json

```diff
@@ -183,13 +177,12 @@
 			"revisionTime": "2018-07-23T16:30:02Z"
 		},
 		{
-			"checksumSHA1": "w5NkRQ2Cebz7a6UfrbwFdhyRWnM=",
 			"path": "github.com/Shopify/sarama",
-			"revision": "46c83074a05474240f9620fb7c70fb0d80ca401a",
-			"revisionTime": "2019-07-22T18:01:16Z",
+			"revision": "355d120d09706d85bc6d2a84287be7fbd13ef453",
+			"revisionTime": "2019-11-22T16:04:21Z",
 			"tree": true,
-			"version": "v1.23.1",
-			"versionExact": "v1.23.1"
+			"version": "v1.24.1-elastic",
+			"versionExact": "v1.24.1-elastic"
```

Comment:
The original PR to `master` has `checksumSHA1` and `origin` fields in the updated object (right side of diff). Is it okay if the backport PR doesn't have these fields?